### PR TITLE
Refactorization/engines

### PR DESCRIPTION
**Pull request by rodacato**
Refactorize logic for workshop engine.

Already reviewed by @vicmaster and @zalapeach

### CODE REVIEW

**Comment by zalapeach** [comment]
@rodacato sure! I will check this @houndci comments, thanks!

**Comment by zalapeach** [comment]
@rodacato I fix all the @houndci comments, I only left one, because I don't know what mean with this comment

**Comment by zalapeach** [comment]
Fix all comments @vicmaster @rodacato ... merge or what?

File: .hound.yml [hunk 1]
```diff
@@ -1,2 +1,7 @@
 CoffeeScript:
   Enabled: true
+StringLiterals:
+  EnforcedStyle: single_quotes
+LineLength:
+  Enabled: true
+  Max: 120
```
Comment (iwano) [inline .hound.yml]: 80

**Comment by vicmaster** [comment]
Please check broken testing.

**Comment by zalapeach** [comment]
The broken testing is already checked, The fix for that is [here](https://github.com/crowdint/kardex-crowdint-com/pull/108)

**Comment by zalapeach** [comment]
@rodacato the links that come from workshops engine are not reachable for the way `workshops_engine.my_link_path`, whats the way to get that routes?

**Comment by rodacato** [comment]
@zalapeach, you should have access like 'my_link_path', the routes are mounted directly on the app, where is failing?, i can check it today

**Comment by zalapeach** [comment]
When I try to go to this route `http://localhost:3000/admin/workshops` the root_path is not available

**Pull request merged**